lib/Controller/FoldersController.php

```diff
@@ -23,65 +23,57 @@
 
 namespace OCA\Mail\Controller;
 
-use OCP\IRequest;
+use Horde_Imap_Client_Exception;
+use OCA\Mail\Contracts\IMailManager;
 use OCA\Mail\Service\AccountService;
 use OCP\AppFramework\Controller;
 use OCP\AppFramework\Db\DoesNotExistException;
-use OCP\AppFramework\Http\JSONResponse;
 use OCP\AppFramework\Http;
+use OCP\AppFramework\Http\JSONResponse;
+use OCP\IRequest;
 
 class FoldersController extends Controller {
 
 	/** @var AccountService */
 	private $accountService;
 
-	/**
-	 * @var string
-	 */
+	/** @var string */
 	private $currentUserId;
 
+	/**  @var IMailManager */
+	private $mailManager;
+
 	/**
-	 * @param string $appName
+	 * @param type $appName
 	 * @param IRequest $request
 	 * @param AccountService $accountService
-	 * @param $UserId
+	 * @param string $UserId
+	 * @param IMailManager $mailManager
 	 */
-	public function __construct($appName, IRequest $request, AccountService $accountService, $UserId) {
+	public function __construct($appName, IRequest $request,
```

Comment:
Really split this into more than one line? If that is the coding-convention here, okay.